server/api/products.js

```diff
@@ -1,10 +1,11 @@
 const router = require('express').Router()
 const Product = require('../db/models/product')
+const Category = require('../db/models/category')
 module.exports = router
 
 router.get('/', async (req, res, next) => {
   try {
-    const products = await Product.findAll()
+    const products = await Product.findAll({include: [{model: Category}]})
```

Comment:
You may want to research and learn about Sequelize Scopes (composable and reusable query options)